ta-sdk-core/src/main/java/com/ibm/ta/sdk/core/util/FreeMarkerTemplateResolver.java

```diff
@@ -144,7 +159,18 @@ public void resolveTemplatesForTargets(List<String> selectedTargets) throws TAEx
     }
 
     public void resolveTemplates(String target) throws IOException, URISyntaxException, TemplateException {
-        String templatesDir = this.middleware+"/templates/"+target+"/";
+        String templatesDir = null;
+        if (this.templateFileDir != null) {
+            templatesDir = this.templateFileDir.getCanonicalPath()+"/";
```

Comment:
Does the trailing "/" need to be a FileSeperator? Or is this inside in a Java flow already?